site/reference/cli/alpha/rpkg/propose-delete/README.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "`del`"
+linkTitle: "del"
+type: docs
+description: >
+Propose deletion of a published package revision.
+---
+
+<!--mdtogo:Short
+    Propose deletion of a published package revision.
+-->
+
+`propose-delete` proposes a published package revision for deletion.
```

Comment:
Should we hold off on the docs update until release?

Reply:
It probably isn't a big deal, the next release should happen late next week I think. But in general we probably want to hold off on these until we have a release.

Reply:
sg! I'll take the docs out of this PR and make a separate one that we can merge at the time of release. Also I realized that we should also have a way for cli users to change the lifecycle from DeletionProposed back to Published, so I will create that in a followup as well.